.renovaterc

```diff
@@ -14,7 +14,7 @@
     "@uirouter/angularjs"
   ],
   "labels": ["dependencies", "javascript"],
-  "reviewers": ["armenzg"],
+  "reviewers": ["sclements"],
```

Comment:
Ah, you beat me to it. I had a pr for this but hadn't merged it yet :) This should be my github handle right? If so, it's 'sarah-clements'.